cql3/statements/create_table_statement.cc

```diff
@@ -229,6 +229,9 @@ std::unique_ptr<prepared_statement> create_table_statement::raw_statement::prepa
     }
 
     stmt->_use_compact_storage = _properties.use_compact_storage();
+    if (stmt->_use_compact_storage && !db.get_config().enable_create_table_with_compact_storage()) {
+        throw exceptions::invalid_request_exception("CREATE TABLE WITH COMPACT STORAGE support is disabled");
```

Comment:
Please point to the config value for enabling it, and also mention that it will eventually be removed.

Reply:
done in v2